mindsdb/integrations/handlers/google_fit_handler/google_fit_tables.py

```diff
@@ -0,0 +1,71 @@
+from mindsdb.integrations.libs.api_handler import APITable
+from mindsdb.integrations.libs.response import HandlerResponse as Response
+from mindsdb.integrations.utilities.sql_utils import extract_comparison_conditions
+from mindsdb_sql.parser import ast
+import datetime
+import pytz
+import time
+from tzlocal import get_localzone
+
+class GoogleFitTable(APITable):
+
+    def time_parser(self, args) -> int:
+        ymd = args.split('-')
+        epoch0 = datetime.datetime(1970, 1, 1, tzinfo=pytz.utc)
+        time = pytz.timezone(str(get_localzone())).localize(datetime.datetime(int(ymd[0].rstrip()), int(ymd[1].rstrip()), int(ymd[2].rstrip())))
+        return int((time - epoch0).total_seconds() * 1000)
+    
+    def select(self, query: ast.Select) -> Response:
+
+        conditions = extract_comparison_conditions(query.where)
+        
+        params = {}
+        filters = []
+        steps = {}
+        now = int(round(time.time() * 1000))
+        one_year = 31536000000
+        one_month = 2629746000
+        for op, arg1, arg2 in conditions:
+            if op == 'or':
+                raise NotImplementedError(f'OR is not supported')
+            if arg1 == 'date':
+                print(f'args : {arg2} 2222222222222222222')
```

Comment:
Can we remove prints please?

Reply:
Sorry about that we pushed some of our debugging print statements accidentally, and I just delete all of them.